### PR TITLE
added password reset functionality

### PR DESCRIPTION
**Pull request by ppietraszko**
Users can now reset their password from the log in page and will receive an email with a temporary password to log in. 

UPDATED:: TEST CASES

### CODE REVIEW

File: templates/changepass/resetPass.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+{% else %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/student_theme.min.css' %}">
+{% endif %}
+<link href="{% static 'changepass/css/changepass.css' %} " rel="stylesheet">
+    <script src="{% static 'js/jquery-3.3.1.min.js' %}"></script>
+    <script src="{% static 'js/popper.js' %}"></script>
+    <script src="{% static 'js/bootstrap.min.js' %}"></script>
+    <script src="https://kit.fontawesome.com/fc24e2383c.js" crossorigin="anonymous"></script>
+    <!-- Custom styles for this template -->
+    <title>oneapply</title>
```
Comment (abe-shek) [inline templates/changepass/resetPass.html]: Having this as OneApply would be consistent

File: logIn/templates/logIn/index.html [hunk 2]
```diff
@@ -38,6 +38,7 @@ <h4 class="h5 mb-3 font-weight-light">to continue, login with your</h4>
                     <p class="errorlist" style="color: red">Invalid credentials</p>
                 {% endif %}
                 <input class="btn btn-lg btn-primary btn-block" type="submit" value="Login"/>
+                <a style="margin:auto;" href="{% url 'changepass:reset_password' user_type %}">Forgot Password?</a>
```
Comment (abe-shek) [inline logIn/templates/logIn/index.html]: Having "class=mt-3" would give this link a bit of top margin and separate it from the login button, "margin: auto" seems to not put any margin to this link rn,

File: templates/changepass/resetPass.html (hunk 1 above)
Comment (ppietraszko) [reply templates/changepass/resetPass.html]: done

File: logIn/templates/logIn/index.html (hunk 2 above)
Comment (ppietraszko) [reply logIn/templates/logIn/index.html]: changing it to have class does not seem to add any extra margin

File: settings/base.py [hunk 3]
```diff
@@ -32,6 +32,7 @@
 INSTALLED_APPS = [
     "landingpage",
     "dashboard",
+    "changepass",
```
Comment (abe-shek) [inline settings/base.py]: Having "changepass.apps.ChangepassConfig" (whatever is the proper name for the app config) will help cover the apps.py for changepass applict.

Comment (ppietraszko) [reply settings/base.py]: done

**Pull request merged**